src/components/Redirect.tsx

```diff
@@ -32,14 +32,14 @@ const Redirect = () => {
     const contract = new ethers.Contract(skillWalletAddress, SkillWalletABI, signer);
     console.log('cntrct: ', contract);
 
+    console.log(discordId);
     const createTx = await contract.addDiscordIDToSkillWallet(discordId);
 
     const res = await createTx.wait();
     const { events } = res;
     const discordIDAdded = events.find((e) => e.event === 'DiscordIDConnectedToSkillWallet');
     if (discordIDAdded) {
-      // return tokenID.
-      alert(`Congrats ${discordId}!`);
+      alert(`Congrats, you've connected your SkillWallet to your Discord ID!`);
```

Comment:
these are from Mike, but in general we should not use alert as they are very old stuff, they block the whole browser and ugly, instead lets use

dispatch(
          openSnackbar({
            message: 'Congrats, you've connected your SkillWallet to your Discord ID!',
            severity: 'success',
            duration: 30000,
          })
        );